docs/source/user-guide/using-virelay-with-gunicorn.rst

```diff
@@ -0,0 +1,20 @@
+===========================
+Using ViRelAy With Gunicorn
+===========================
+
+As described in the :doc:`../getting-started/basic-usage` article, ViRelAy provides a command line interface, to start the using a simple built-in HTTP server. This HTTP server is, however, rather slow, because it is limited to responding to requests in sequence. To unlock the full potential of ViRelAy or even run it on a remote server to collaborate with others, it is recommended to run ViRelAy using the WSGI HTTP server `Gunicorn <https://gunicorn.org/>`_. Gunicorn was installed together with ViRelAy, but if you want to use a specific version, it can also be installed from PyPi: ``pip install gunicorn``.
```

Comment:
```suggestion
As described in the :doc:`../getting-started/basic-usage` article, ViRelAy provides a command line interface to serve the application using a simple built-in HTTP server. This HTTP server is, however, rather slow, because it is limited to responding to requests in sequence. To unlock the full potential of ViRelAy or even run it on a remote server to collaborate with others, it is recommended to run ViRelAy using the WSGI HTTP server `Gunicorn <https://gunicorn.org/>`_. Gunicorn was installed together with ViRelAy, but if you want to use a specific version, it can also be installed from PyPi: ``pip install gunicorn``.
```

Reply:
Accepted